isr_exploration/CTE_Thurs.ipynb

```diff
@@ -0,0 +1,332 @@
YdyGdsj7hSXsat8suEYW43suRsn9KRXI8kR7l7L01wxuis/7s5g2e5Mlmw7yfkDk5k+vLO3h+XXtGf5BfDXKQNYdyiXPelFvLwilTN7JXBG74QWv25qVjFz3lhr607w0NQBzD2zR4P7Dw/NZNG1Z7Jw4zGe/HYvuSWVrD6Yw4XPr2TuGd25Y1IfokJat5dzS/DpxmPsSFMlFP8xLI+Ancpi15CVrpYM297EyQfOgJ8eU+93fgET73d9gOUF8Mk1UF0OwqRq4GXuwBzXA3JT6ShyubvwKe6Yl8xb142rVQhf07roNmHNpHtCOK9cdRpmk6Cy2sJNH2zkZEFZi1xr6baTzF+nrFljesR5pDNCdJhSTO85v6+tlRDAtuMFfLE5rU01b84vrbQlGnSMDuE2V5IjfMCsrtrxDLV1MXnoix2k5bfMvaTRgArHeOnKEYQEmpAS7lywhZwW7hRwOLuEK99YY+tIcP/k/tw4oadte0PWJ5NJcPnoFJbfM5G5Z3THJJRl661fD3Huf37m043HvWbd9oS3obC8iqeXbAWgY6iFGcKoe20Orh0T1xDDr4SzH1Cv9ZHQGzqoJD92NsMALSV8eQvkGgkR5zwIoSoWmIgOyCGXATDevJMRh17TSV9exi2lTgjRXwgxXwhxUghRKYQ4zVj/sBDiHM8M0Xc5o3cCjxiN2rOKKrjp/Y2UVdZ41K14LLeUBz7fBkBMWCDPXzHcY7Mgk0lw23l9eP/6MbYuFBXVFu5csIUpL6xk+Z7MNuEK/I9DcsRDU11MjvARs3piZDBPXmqPdbr3k61tSrFui7R3+dU7KZJHLhoEQGZRBX/+VLURa4n4tGO5pVz5xhpb1v7d5/fl/ybWdis2lOhlJToskEcuHsTS2yfY+lV37JQAACAASURBVJFmF1dw78Kt7Drpna457rh9rd/zrR9tIrtCyd+Hor8hYO9itUO/CyHUCe/PiKvgnL+o14YYNFO9Zu2BTBc7I63+H+w2xtTndzD+bvs2IRDTnsVidK+4I2AR+379nM/aqBvcH2i2UieEGA6sB84GVgCOhdIigD+6NbI2wjWnd+fqcarDxPa0Au77bBsLNxzzSHxHdY2FOz7ebMsOe/LSoXSMDnV7zHWZ0CeRoUbcSqBZKYx70ou47t31XP76GjYeyfX4NUEpkECzCiJbBeKzP+zlI6P10fjeCUwZ4tvJEY0xaWCyLUN59cEc3vr1kJdH5L9o+aW4fHRXpg7tCMBPezJ5+7fDHo9PS8svY84bazhh1Me8/dzetnI+zWFAxygW3DSO568YTnKUSq6wlkTZnlbAO78dIq8NNJ+3fs8rjaz30WIPU0zroMrISG7I8tYcHAsRu5IwcWQ1/PB39T4mBWa8emombnAEpss/wBKoiiM/G/gyLy76ic1H89wctKY5uFPv4QlgG3A+UAlc7rBtHXCpG+f2OrNGdmFcz3i6xDatRD180SBSM0tYfTCHxVtPOHWMMzz/4342GW3ArhnXjd8NajmFxWQkFQzrEsPEfom89vNBiiqqWXcol0tfWc2kAcncN7kffY2aeFYWbjjG8bwyusSGNhmrJqVk54lCftiVwY97MmwxJJuP5TPx6eWM6h7H6O6xjOoeR8+E8EYze63xeOHBZqeSI9oKD00dyKrUHI7klPL0d3uZ0DfB7ZZOrvyPPHmsO7TCdduP/Gok5kq5/oew9Vg+x/PKeOKb3fSr8xt3h/SCcua8vobjeSqc4I9n9+Ku8/u6fV4hBNOHd+a8Acm8+NN+Xv/5IBLVPefRxbt4/Os9TBqYxOxRXTmrT2Kj3g1v3eNWJCCQPBz4PqLUUITCE6HXuZ67SFxP6DgcTm5RSt3Ev6hWYps/gvyjSmGra+krzoSFc0HWgDkIZr8HYQ3Ecif2w3Txi/DZDcSKYp4zPcct7yfy2W3n0CE6xK2he/v/0xy8OWZ3lLozgaullKVCiLq9QjKAtmsywbVg+kCziZevOo3p//uNo7mlNgHWnBp21puhoqqG14yijv2SI3lw6oAmjvQMZpPg1nP7cNXYbry84gDvrT5CZbWFZbuVIjZzRBfuOr8PXWLVrKypZIfyqhpWp+ao43dnkl5YfzeLwzmlHM4ptVkH4sKDGNUtltHd4xjVPZZBnaIJCjjVsFxSoToxXD++hy0j2SWqjTgiH3EzhwcH8Mxlw5n96ioqayzc+fEWvrz1TIIDmt+Ox52EFHeOdUeweTKJpgHaj/xqIpA+KiSQF+aMYParq6mqkRzIdChN0dhDvx4c/+dn901kzhtrOGrUwrtxfA/un9zPuYlXtXOWtojgAP5y4QDWHsxhy7ECggNMVFRbqKyx8PX2dL7enk6HqBBmntaZ2aO62mrhOeLqvWaxSA7llLAjrcD2t6UXlvP19pMkRgaTGBFMQmQw4UHmRv9Wx/JFl0fvZnDFYbCKxyGXgdnDCSCDZiilLnsfZO6C5EEq7OTIr9BtfO3/b001fHo9FKerzxc+CZ1Pa/z8Q2bBsbWw7nWGm1L5Q/nb3PxBJAtuPp3FW0/YZYH5F5fuKW/JoFrHunRkq8ivBnFHqROoGW59xAItG3XrY8SGB/HWtaOY8fIqmytgw5E8Ln7pV8Z0j2NMjzhGd48j1giGbwjrzRBoFkgJwQEmXrzy1DZgLU1seBAPTh3IdWf24Lll+/h043EsEj7bdJzFW09w9bhuDXZryCqqYPmeTJbtzmDl/mzK6mmBNaBjFIVllaTll9MlNpTeSRFsPJxHkfHd5ZZU8v2uDL7flQGodmjDu8Ywurv6HiscztkhKoTbz22GO2fbJ3BiozHoXVCa2/BMtBUZ2S2WW8/pzQs/HWBPehH//X4ffZIi2txs1ZuCzQm0/HLgtJRY7rmgL099u5dyIxyiqsbS8EO/Aaz/89NSYnj1u80cKlJya+4Z3Xlw6oCmFTpLDax8BtLWqc8lzhW1tU56hnWJ5u4L+vHJhmN8sz2dsqoa0gvLeXlFKi+vSGVM9zhmjerC1
```

Comment:
This seems like it is worth further inspection, and might bring the two results into better agreement